common/util/web_controller/autoware_web_controller/CMakeLists.txt

```diff
@@ -1,12 +1,13 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.5)
 project(autoware_web_controller)
 
-find_package(catkin REQUIRED)
+find_package(ament_cmake REQUIRED)
```

Comment:
Is there a reason why `ament_cmake_auto` was not used here?

Reply:
I don't mind either way, but I was just applying comment from the related PR. https://github.com/tier4/Pilot.Auto/pull/104#discussion_r533314002